src/sentry/static/sentry/app/components/events/interfaces/frame.jsx

```diff
@@ -145,6 +145,18 @@ const Frame = React.createClass({
     if (data.inApp) {
       title.push(<span key="in-app"><span className="divider"/>{t('application')}</span>);
     }
+
+    if (!title.length) {
+      title = [<code key="title">{'<invalid frame>'}</code>];
+      if (defined(data.errors)) {
+        title.push(
+          <a key="frame-errors" className="in-at tip original-src" data-title={data.errors.join(', ')}>
```

Comment:
The only thing shitty here is that this tooltip requires being clicked to open and close since the TooltipMixin defines this behavior in this component. I tried adding another TooltipMixin within another selector `.tip.noclick`, but React didn't like that. :(